test/StringCalculatorKata.Tests/StringCalculatorTests.cs

```diff
@@ -0,0 +1,22 @@
+using Xunit;
+
+namespace StringCalculatorKata.Tests;
+
+public class StringCalculatorTests
+{
+    [Theory]
+    [InlineData("", 0)]
+    [InlineData("5", 5)]
+    [InlineData("1,2", 3)]
+    [InlineData("1,2,3,4", 10)]
+    [InlineData("1,2\n3", 6)]
+    [InlineData("//:\n1:2:3", 6)]
```

Comment:
what happens if you pass bad input? lets add more and more cases to make this fail.

Reply:
try
1,222,r,r
1,,,2
4,3,\n5
7,6\n,3,2\n

and more like this